python/lsst/ts/mtrotator/rotator_csc.py

```diff
@@ -494,6 +533,7 @@ async def telemetry_callback(self, client):
                 client.telemetry.latching_fault_status_register,
                 client.telemetry.latching_fault_status_register_axis_b,
             ],
+            copleyFaultStatus=client.telemetry.copley_fault_status_register,
```

Comment:
Please, consider adding some level of backward compatibility. See my previous comment.

Reply:
Fixed in the commit: [f03b120](https://github.com/lsst-ts/ts_mtrotator/pull/82/commits/f03b120f7da830090949ea0b1a365e87fd3689b1). Thanks!